.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
We need this for the child process to be able to resolve `nock` (I think the alternative would be to bundle `recordRequests`, however changing the linker seemed easier to implement).

Reply:
The child process should be able to resolve it without changing the linker, what error are you getting?

Reply:
Yeah that seems to be correct, not sure what I did to see the error (I probably ran `node` instead of `yarn node` in my manual tests) but yeah the tests results are the same regardless of the linker. Let me revert that.